R/germline.R

```diff
@@ -93,19 +76,58 @@ take_first_allele <- function(string) {
   unlist(strsplit(string, ","))[1]
 }
 
-generate_germline_sequence <- function(v_seq, j_seq) {
-  if (is.na(v_seq) || is.na(j_seq)) {
+generate_germline_sequence <- function(seq, v_ref, v_end, cdr3_start, cdr3_end, j_start) {
+  if (is.na(seq) || is.na(v_ref) ||
+    is.na(v_end) || is.na(cdr3_start) || is.na(cdr3_end) || is.na(j_start)) {
```

Comment:
Wouldn't something like `if(any(sapply(c(seq, v_ref, ....), is.na)) { #do.smth }` work better?